<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConcurentPrijs</class>
 <widget class="QDialog" name="AddConcurentPrijs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voer prijzen concurentie in.</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbNaam">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>35</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Naam:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbNaam">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>30</y>
      <width>356</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>tbNaam</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStraat">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Straat:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbStraat">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>55</y>
      <width>356</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>tbStraat</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEMail">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbEMail">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>105</y>
      <width>356</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>tbEMail</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNummer">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>61</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Nummer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbNummer">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>5</y>
      <width>356</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>tbnummer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoek">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>5</y>
      <width>61</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/zoom.png</normaloff>:/icons/icons/zoom.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lbGemeente">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gemeente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbGemeente">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>80</y>
      <width>356</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>tbGemeente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWis">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>61</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/draw_eraser.png</normaloff>:/icons/icons/draw_eraser.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Afsluiten zonder opslaan</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/stop2.png</normaloff>:/icons/icons/stop2.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lbArtNr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>15</y>
      <width>236</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Art.Nr.:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Aanpassen</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/edit_button.png</normaloff>:/icons/icons/edit_button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNew">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Nieuw</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Opslaan</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/save_close.png</normaloff>:/icons/icons/save_close.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>56</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wissen</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/cross.png</normaloff>:/icons/icons/cross.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>185</y>
     <width>541</width>
     <height>36</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbDatumAankoop">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>116</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum Aankoop:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPrijs">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prijs:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtpDatumAankoop">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbPrijs">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>5</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tbPrijs</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tvPrijzen">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>220</y>
     <width>541</width>
     <height>486</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
